search/src/Components/wrapper/reactive_search.js

```diff
@@ -557,11 +558,41 @@ class ReactiveSearchComponent extends Component {
         return (
             <ReactiveBase
                 app="weedid"
+                mapKey=""
                 url={baseURL + "elasticsearch/"}
                 theme={theme}
                 headers={{'X-CSRFToken': Cookies.get('csrftoken')}}
             >
-                <div style={{ position: "fixed", width: "20rem", overflow: "scroll", height: "90%", left: 0, padding: '0 1rem' }}>
+                <div style={{ position: "fixed", width: "20rem", overflow: "scroll", height: "90%", left: 0, padding: '1rem' }}>
+                    <GeoDistanceSlider
+                      title="Location"
+                      componentId="gds_filter"
+                      placeholder="Search Location"
+                      dataField="location"
+                      unit="km"
+                      URLParams
+                      react={{
+                        and: [
+                          "crop_type_filter",
+                          "category_filter",
+                          "grains_text_filter",
+                          "task_type_filter",
+                          "lighting_filter",
+                        ],
+                      }}
+                      range={{
+                        start: 10,
+                        end: 1000
```

Comment:
Please ask the client what an appropriate max range is if we are forced to use a radius approach.

Reply:
@geezacoleman where do you think should be the default centre and radius of the location filter? Now I'm setting it as Sydney and the default radius range to be 10-1000 kms.

Reply:
(thinking about this, if we want all datasets to be searchable initially, we need to make the radius cover the whole globe?)

Reply:
I think what @jnothman suggests is probably a good idea, would ideally need to take a global perspective first and then allow people to zoom in. Gives a nice representation of dataset distribution too. Other option is to randomly select a dataset and use that as the default location each time page is refreshed? But the first option makes more sense.